doc/pal.md

```diff
@@ -0,0 +1,89 @@
+# PAL
+================
+
+## Overview
+
+The Platform Abstraction Layer interfaces presented here are:
+
+- Execution engine
```

Comment:
We should have a task at least to fill this in with the other modules that you've brought in.